static/app/components/teamSelector.tsx

```diff
@@ -46,6 +48,20 @@ const unassignedOption = {
   disabled: false,
 };
 
+const CREATE_TEAM_VALUE = 'CREATE_TEAM_VALUE';
```

Comment:
I wanted to use `Symbol()` at first to guarantee a unique value but react-select cannot handle it...
However, the current value should never clash with any team slug.